src/ApiCore/Serializer.php

```diff
@@ -400,4 +427,16 @@ private static function getPhpArraySerializer()
         }
         return self::$phpArraySerializer;
     }
+
+    public static function loadKnownMetadataTypes()
+    {
+        foreach (self::$metadataKnownTypes as $key => $class) {
+            new $class;
```

Comment:
I don't know of a *cleaner* way. I tried using `class_exists`, but that does not load the class into the descriptor pool. The problem is that we need this line to be executed from the message class: https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Container/src/V1/AutoUpgradeOptions.php#L36. And that will only be executed if we call the constructor.

A different way of doing it would be to call the metadata class's `initOnce` method explicitly. I'm not sure if that is a good idea though, as right now we don't ever refer directly to the metadata classes.